lib/galaxy/objectstore/templates/examples/production_onedata.yml

```diff
@@ -0,0 +1,54 @@
+- id: onedata
+  name: Onedata Storage
+  version: 0
+  description: |
+    This template allows using Onedata storage.
```

Comment:
Proponuję

If you have an account in Onedata, you can use such an object store as a storage location for your Galaxy datasets; they will be stored in the Onedata space of your choice.

Reply:
to More on z linkiem zostawić?

Reply:
tak, to zostaje, tylko wymieniamy pierwsze zdanie

Reply:
EDIT zaktualizowałem o wymaganie wersji zona